spec/system/task_spec.rb

```diff
@@ -1,24 +1,24 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+  let(:task) { create(:task, project_id: project.id) }
```

Comment:
Factoryに`association`をつけると省略可能となります。
```suggestion
  let(:task) { create(:task) }
```

Reply:
444e5f2で修正しました。